disque.go

```diff
@@ -9,17 +9,18 @@ import (
 	"github.com/garyburd/redigo/redis"
 )
 
-// Conn represent connection to a Disque Pool.
-type Conn struct {
-	pool *redis.Pool
-	conf Config
+// Pool represent Redis connection to a Disque Pool
+// with a certain Disque configuration.
+type Pool struct {
+	redis *redis.Pool
+	conf  Config
 }
 
-// Connect creates new connection to a given Disque Pool.
-func Connect(address string, extra ...string) (*Conn, error) {
+// New creates new connection to a given Disque Pool.
+func New(address string, extra ...string) (*Pool, error) {
```

Comment:
I will think about it, maybe I'll reconsider. It's not quite common method yet. But it's elegant, I agree.

Thanks for all your useful feedback, Fabrizio!